rigs/heft-node-rig/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@rushstack/heft-node-rig",
+  "version": "0.0.0",
+  "description": "A Heft rig for building Node projects.",
```

Comment:
This description misleadingly implies that rigs are  Heft concept.

`A rig package for Node.js projects that build using Heft`